### ISSUE TITLE
Update eslint to latest version

### ISSUE THREAD

**Issue by Plastikmensch** [Krizzu/eslint-check-action#12]
Is there a reason this action still uses eslint 6.6.0?

**Comment by krizzu** [comment]
Haven't touched it in a while, I'm happy to accept any PR :)

**Comment by Plastikmensch** [comment]
Thanks for the response
I could try, but I've never done that before

**Comment by Plastikmensch** [comment]
What's your opinion of creating a dependabot config, which only allows updates of eslint?
I noticed that updating other dependencies breaks this action and I'm not familiar with typescript,
so I don't really want to make changes to the codebase.

**Comment by krizzu** [comment]
Hey,

> I noticed that updating other dependencies breaks this action

which dependencies? I could take a look later this week with upgrading eslint to the latest version (need to check what breaking changes v7 has)

> I'm not familiar with typescript,
so I don't really want to make changes to the codebase.

Don't worry, try to do small changes and see what happens. You can always open a draft PR so we can collaborate there.

You can test this action locally (run it on your machine) - it's not easy to setup, though. Better option would be to
create a test repo and install action from your fork directly and run it on PR ([like I did this in the beginning](https://github.com/Krizzu/gh-actions-tester/blob/master/.github/workflows/eslint.yml))

**Comment by Plastikmensch** [comment]
> which dependencies? I could take a look later this week with upgrading eslint to the latest version (need to check what breaking changes v7 has)

Updating the core and github dependency threw a TypeError, but I don't have the log anymore, because I re-ran the workflow...
> Don't worry, try to do small changes and see what happens. You can always open a draft PR so we can collaborate there.

Seems like a good idea, but I'm not sure how much I can contribute
The only thing I can say for now is that bumping eslint to 7.7.0 worked for me.
Would create a new branch from this repos master before opening a draft though, because I already made changes to my forks master and I prefer a clean commit history,
or if two useless commits don't bother you, I would just create a new branch on my fork for the PR
> You can test this action locally (run it on your machine) - it's not easy to setup, though. Better option would be to
> create a test repo and install action from your fork directly and run it on PR ([like I did this in the beginning](https://github.com/Krizzu/gh-actions-tester/blob/master/.github/workflows/eslint.yml))

Yeah creating a test repo is a good idea :D
Right now I'm testing with an actual repo and the commit history is just full of workflow stuff

**Comment by krizzu** [comment]
> Would create a new branch from this repos master before opening a draft though, because I already made changes to my forks master and I prefer a clean commit history,

Coolio, I prefer that too. Good tip is to always create a feature branch, even if working on a fork. Let me know if you need any help

**Comment by Plastikmensch** [comment]
I see you use semantic commit messages, so bumping eslint falls under "chore:", right?
The PR title should probably be semantic too, like "chore: Updating Dependencies", or would you prefer another title?

**Comment by krizzu** [comment]
Yes, version bump is `chore` work. I'll leave the title up to you - whatever you feel is right.
